### ISSUE TITLE
Support quoted attributes in DOT import

### ISSUE THREAD

**Issue by metasoarous** [gephi/gephi#1195]
Some libraries (have https://github.com/aysylu/loom in mind) export node attribute names quoted. Gephi only accepts quoted attribute names. It would be very helpful to allow for quoted attributes.

**Comment by mbastian** [comment]
In which format? GEXF? Can you give an example of such a file? Thanks!

**Comment by metasoarous** [comment]
My apologies; dot format specifically. Here's a simple example:

```
digraph "thegraph" {
  A -> B
  B -> C
  A -> C
  A ["origin"="foo"]
  B ["origin"="bar"]
  C
}
```

Remove the quotes around `origin`, and the attribute loads fine, but not otherwise.

**Comment by mbastian** [comment]
Legitimate, and easy to fix

**Comment by mbastian** [comment]
Fixed in trunk, will be included in upcoming 0.9 release.

**Comment by metasoarous** [comment]
Awesome; thanks!
